pkg/dbmate/db_test.go

```diff
@@ -273,45 +547,13 @@ func TestMigrate(t *testing.T) {
 			require.NoError(t, err)
 			defer dbutil.MustClose(sqlDB)
 
-			count := 0
-			err = sqlDB.QueryRow(`select count(*) from schema_migrations
-				where version = '20151129054053'`).Scan(&count)
+			// check applied migrations
+			appliedMigrations, err := drv.SelectMigrations(sqlDB, -1)
 			require.NoError(t, err)
-			require.Equal(t, 1, count)
-
-			err = sqlDB.QueryRow("select count(*) from users").Scan(&count)
-			require.NoError(t, err)
-			require.Equal(t, 1, count)
-		})
-	}
-}
-
-func TestUp(t *testing.T) {
-	for _, u := range testURLs() {
```

Comment:
As dumps database specific we need to have dump for each database or perform this test on a known base (I chose this option)